### PR TITLE
 [BEAM-3114] Generate text proto config properly in container boot code

### PR DESCRIPTION
**Pull request by herohde**
Also disable stack trace optimization in java container.

### CODE REVIEW

**Comment by herohde** [comment]
R: @tgroh

**Comment by herohde** [comment]
retest this please

File: sdks/java/container/boot.go [hunk 1]
```diff
@@ -20,19 +20,20 @@ package main
...
 	"log"
 	"os"
 	"path/filepath"
 	"strconv"
 	"strings"
 
 	"github.com/apache/beam/sdks/go/pkg/beam/artifact"
-	pb "github.com/apache/beam/sdks/go/pkg/beam/model/fnexecution_v1"
+	fnpb "github.com/apache/beam/sdks/go/pkg/beam/model/fnexecution_v1"
+	pb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
```
Comment (tgroh) [inline sdks/java/container/boot.go]: why this selection for name, out of curiosity?

Comment (herohde) [reply sdks/java/container/boot.go]: I don't have a better one for the basic data structure package.

**Comment by herohde** [comment]
@tgroh - is there anything you would like me to change?

File: sdks/java/container/boot.go (hunk 1 above)
Comment (tgroh) [reply sdks/java/container/boot.go]: I'd prefer something like `pmpb`, but if you feel strongly I'm not extremely concerned

Comment (herohde) [reply sdks/java/container/boot.go]: Ack. I prefer just pb.

**Pull request closed without merging**